src/PotentialTypes/BasisPotentials/SNAP/utilities/utl.jl

```diff
@@ -68,26 +67,23 @@ function neighborlist(A::AbstractSystem, snap::SNAPParams)
     for n = 1:length(X)
         n_element = findall(x->x==Symbol(A[n].atomic_symbol), elements)[1]
```

Comment:
Replace `Symbol(A[n].atomic_symbol)` with `atomic_symbol(A, n)` -- depending on the concrete implementation of `A` the current version may be inefficient or not work entirely; the result of the function is already a symbol, so no need to convert it.